Gruntfile.js

```diff
@@ -112,15 +112,15 @@
 					command: [
 						"cd build/firmware",
 						"unzip UI.zip",
-						"rsync -azp --chmod=Du=rwx,Dg=rx,Do=rx,Fu=rw,Fg=r,Fo=r * <%= secrets.firmware.opensprinkler.location %>",
-						"rsync -azp --chmod=Du=rwx,Dg=rx,Do=rx,Fu=rw,Fg=r,Fo=r * <%= secrets.firmware.betaui.location %>"
+						"firebase deploy --only hosting:opensprinkler-betaui",
+						"firebase deploy --only hosting:opensprinkler-ui"
```

Comment:
This should move to a GitHub Action allowing auto deployment to beta on merge and a click to prod deployment.